versions.gradle

```diff
@@ -20,6 +20,7 @@ ext {
     datadog_version = '2.11.0'
     logback_version = '1.2.3'
     javax_annotation_version = '1.3.2'
+    os_locator_sdk_version = '1.0-SNAPSHOT'
```

Comment:
oh, @scirner22 this was the other thing I am waiting on, I have [this ticket in](https://app.clubhouse.io/figure/story/102372/upgrade-ossrh-secrets-from-p8e-repository-level-to-provenance-io-organization-level) to get the OSSRH secrets moved to the org level so I can publish the os-locator-sdk to Maven Central... I'll see if they will actually get to it today